src/test/java/uk/ac/ebi/atlas/bioentity/properties/ExpressedBioentityFinderImplIT.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.ebi.atlas.bioentity.properties;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+import uk.ac.ebi.atlas.configuration.TestConfig;
+
+import javax.inject.Inject;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.ac.ebi.atlas.testutils.RandomDataTestUtils.generateRandomEnsemblGeneId;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = TestConfig.class)
+@WebAppConfiguration
+class ExpressedBioentityFinderImplIT {
+    @Inject
+    private ExpressedBioentityFinderImpl subject;
+
+    @Test
+    void unknownGeneIdIsNotExpressed() {
```

Comment:
I think this is a negative test case if understood the code correctly :). It would be nice if you add a  test case to test the positive scenario.

Reply:
Good catch!